flow/Trace.cpp

```diff
@@ -884,6 +895,28 @@ TraceEvent& TraceEvent::suppressFor( double duration, bool logSuppressedEventCou
 	return *this;
 }
 
+TraceEvent& TraceEvent::setMaxFieldLength(int maxFieldLength) {
+	if(maxFieldLength == 0) {
+		this->maxFieldLength = FLOW_KNOBS ? FLOW_KNOBS->MAX_TRACE_FIELD_LENGTH : 495;
```

Comment:
Hm, there are instances where `FLOW_KNOBS` might not be initialized?

Reply:
I was thinking there was, but looking closer it seems like maybe you'd have to log trace events in static initialization or something to trigger it. There is a very small precedent for checking the existence of `FLOW_KNOBS` in memory sampling and assert checking.

Reply:
Okay, I see. Well, I suppose this isn't hurting anything.